### PR TITLE
Added Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by luxdev01**
I translated the `strings.xml`, `full_description.txt`, `short_description.txt` and `title.txt` files into Brazilian Portuguese.

### CODE REVIEW

**Comment by nain-F49FF806** [comment]
Hi, Super excited to see your translation PR.
I see you have also done the description and title files. That is amazing!

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!--
+  ~ Copyright (c) 2021 2bllw8
+  ~ SPDX-License-Identifier: GPL-3.0-only
+  -->
```
Comment (nain-F49FF806) [inline app/src/main/res/values-pt-rBR/strings.xml:4]: As the translation is provided primarily by you, I think we can do away with this copyright notice. Feel free to remove it entirely, or add your name and current year.

**Review by nain-F49FF806 — CHANGES_REQUESTED** [review]
The rest lgtm! 

As I don't speak the language, I am content with you being satisfied with the translation you've provided. :)

Just consider doing the little change requested, and we can merge it in.

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 1 above)
Comment (luxdev01) [reply app/src/main/res/values-pt-rBR/strings.xml]: Resolved, and thanks!

**Comment by nain-F49FF806** [comment]
Thank you for your contribution! 
This shall be part of the next release.

**Pull request merged**